ivy/functional/backends/jax/experimental/losses.py

```diff
@@ -64,26 +64,6 @@ def soft_margin_loss(
         return loss
 
 
-def kl_div(
```

Comment:
You just removed the backend implementation from jax? 🤔 I am not following what you are trying to do. Can you elaborate more by providing description in the PR?

Reply:
We don't need a backend specific in these cases because there is no efficient implementation of it, so even if we define a compositional implementation, the function will fall back on to that (the one defined in `ivy/experimental/losses.py`). If you'll take a look at the previous implementations, they were using a compositional approach only, there is no primary function that exists for `numpy`, `jax`, `tensorflow`

Reply:
We can define a compositional implementation in `ivy/experimental/losses.py`, so that for all the backends not having a primary or efficient implementation, this compositional implementation is called and we don't have to write the same function in each backend

Reply:
We still need to implement backend-specific implementation 😅 . Even if a backend doesn't provide the implementation we still provide that (as the composition of other ivy functions) because other backends are providing it.

Reply:
Got it! So only Torch has an implementation of k1_div in its functional API. 😄 Looks good to me.